pynecone/utils.py

```diff
@@ -500,7 +500,10 @@ def export_app(app: App, zip: bool = False):
 
     # Zip up the app.
     if zip:
-        cmd = r"cd .web/_static && zip -r ../../frontend.zip ./* && cd ../.. && zip -r backend.zip ./* -x .web/\* ./assets\* ./frontend.zip\* ./backend.zip\*"
+        if os.name == "posix":
+            cmd = r"cd .web/_static && zip -r ../../frontend.zip ./* && cd ../.. && zip -r backend.zip ./* -x .web/\* ./assets\* ./frontend.zip\* ./backend.zip\*"
+        if os.name == "nt":
```

Comment:
Would the check `platform.system() == "Windows"` work? This is what we use in other parts of the code, so would be good to stay consistent. We should probably just move this to `utils.py` as a function `def is_windows` or something later.

Reply:
I also thought about that. The differences between os.name and platform.system() is that the os provides for Linux and OSX the value posix and for Windows the value nt. On the other side, platform.system provides for Linux  the value Linux, for OSX the value Darwin and for Windows Windows. That produce an additional check in the if statement. We can harmonise it to use platform.system if you think it is better. But I don't recommend to introduce a new is_windows function because the feature is already in os.name or platform.system.

Reply:
That makes sense - I think in other places we just put the Linux/OSX within the `else` case so we don't do an explicit check. But this looks good to me, let's ship it!